components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/java/org/wso2/carbon/identity/api/server/configs/v1/core/ServerConfigManagementService.java

```diff
@@ -844,15 +842,7 @@ private UserDefinedLocalAuthenticatorConfig castToUserDefinedConfig(LocalAuthent
     private void resolveEndpointConfiguration(Authenticator authenticator, UserDefinedLocalAuthenticatorConfig config) {
 
         UserDefinedAuthenticatorEndpointConfig endpointConfig = config.getEndpointConfig();
-
-        AuthenticationType authenticationType = new AuthenticationType();
-        authenticationType.setType(AuthenticationType.TypeEnum.fromValue(
-                endpointConfig.getAuthenticatorEndpointAuthenticationType()));
-        authenticationType.setProperties(new HashMap<>(
-                endpointConfig.getAuthenticatorEndpointAuthenticationProperties()));
-
         Endpoint endpoint = new Endpoint();
-        endpoint.setAuthentication(authenticationType);
         endpoint.setUri(endpointConfig.getAuthenticatorEndpointUri());
```

Comment:
Why this section is removed ? Shouldn't we set the properties of local authenticator config authentication ?

If this is about two operations like while retrieving we don't need to set etc better have two methods
or should at least denote why this is not needed

Reply:
This is for removing the setting authentication configuration of the action endpoint for  the user defined local authenticator.
<img width="654" alt="Screenshot 2025-01-26 at 20 37 55" src="https://github.com/user-attachments/assets/1421e4ed-1be0-45d5-a4f6-3818ca23a137" />

Here the properties of the action endpoint configuration msut not send in the response payload.